MangoAPI.Client/src/app/components/chats/chats.component.ts

```diff
@@ -90,6 +96,7 @@ export class ChatsComponent implements OnInit {
     }
 
     this.userId = tokens.userId;
+    this.userData = tokens;
```

Comment:
This variable can be skipped because tokens can be got from `_tokensService` directly